docs/content/preview/yugabyte-platform/create-deployments/connect-to-universe.md

```diff
@@ -21,13 +21,36 @@ You can connect to the database on a universe in the following ways:
 
 ## Download the universe certificate
 
-If the universe uses encryption in transit, to connect you need to first download the universe TLS root certificate. Do the following:
+If the universe uses Client-to-Node encryption in transit, to connect you need to first download the universe TLS certificate. Do the following:
 
 1. Navigate to **Configs > Security > Encryption in Transit**.
 
-1. Find the certificate for your universe in the list and click **Actions** and download the certificate.
+1. Find your universe in the list.
 
-For more information on connecting to TLS-enabled universes, refer to [Connect to clusters](../../security/enable-encryption-in-transit/#connect-to-clusters).
+1. Download the certificate.
+
+    - If you are connecting using a YSQL client (such as ysqlsh), click **Actions**, and choose **Download YSQL Cert**.
+
+        This downloads the `yugabytedb.crt` and `yugabytedb.key` files.
+
+    - If you are connecting using a YCQL client (such as ycqlsh), click **Actions**, and choose **Download Root CA Cert**.
+
+        This downloads the `root.crt` file.
+
+    - If you are connecting to universes that are configured with custom CA-signed certificates, obtain the root CA and client YSQL certificate from your administrator. These certificates are not available on YugabyteDB Anywhere for downloading.
+
+1. For connecting using a `ysqlsh` client, paste the `yugabytedb.crt` and `yugabytedb.key` files into the `<home-dir>/.yugabytedb` directory and change the permissions to `0600`, as follows:
```

Comment:
I think we can delete this part because it relates to using the client cert for auth, which is not commonly used. In general, to connect with ysqlsh to a tls enabled cluster, the user doesn't need any certs. More details in the above blog post.